core/src/main/java/org/apache/accumulo/core/client/rfile/RFileWriterBuilder.java

```diff
@@ -59,15 +62,22 @@ OutputStream getOutputStream() {
   }
 
   private OutputArgs out;
-  private SamplerConfiguration sampler = null;
   private Map<String,String> tableConfig = Collections.emptyMap();
   private int visCacheSize = 1000;
+  private Map<String,String> samplerProps = Collections.emptyMap();
+  private Map<String,String> summarizerProps = Collections.emptyMap();
+
+  private void checkDisjoint(Map<String,String> props, Map<String,String> sampleProps, Map<String,String> summrizerProps) {
```

Comment:
That would be the right thing to do.  Its a bit annoying because its private in classes that in the public API space.  So I would need to place in some third class that not in a public API package, and I don't know where to put it.  I will added a comment to both methods mentioning its doppelganger .